### DISCUSSION TITLE
Arrival and Departuretime Optional

### DISCUSSION THREAD

**Discussion by ChillLP** [Traewelling/traewelling#2500] [Ideas / Feature Requests]
I dont know wheter its possible, but it would be nice if i wouldnt have to add and guesstimate Passtrough times or rather Arr/Dep Times at every Intermediate Stop, as i am mostly checking in Cargotrains.

I just wanna use "Stopovers" as markers so my Map is actually correct-ish

**Comment by pascalwittler** [comment]
Some kind of checkbox or toggle that says "This is not a scheduled stop, this is just a waypoint" would be great.

**Comment by ChillLP** [comment]
Exactly my thought
